### PR TITLE
Create German CFP

### PR DESCRIPTION
**Pull request by mfherbst**

### CODE REVIEW

File: 2022/cfp_de.md [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+Jahr ist, in welchem die JuliaCon Vorschläge für nicht-englischsprachige Workshops
+akzeptiert, sollten nicht-englischsprachige Workshops eher einen einführenden
+Charakter haben.
+
+### Virtuelle Poster
+Da die JuliaCon in diesem Jahr eine reine Online-Konferenz ist, werden die
+Posterpräsentatoren keine physischen Poster vorbereiten. Stattdessen werden sie
+gebeten, dreiminütige Aufnahmen ihrer vorgeschlagenen Themen zu erstellen. Die
+Videos werden dann in speziellen Chaträumen zur Verfügung gestellt, sodass
+die neue Teilnehmer im Chatraum sich das Video schnell ansehen und sich direkt an
```
Comment (felixcremer) [inline 2022/cfp_de.md:86]:
```suggestion
die Teilnehmer im Chatraum sich das Video schnell ansehen und sich direkt an
```

Comment (mfherbst) [reply 2022/cfp_de.md:86]: Thanks for the suggestion.

The English original also has "attendees who join the room" which I found a bit clunky to keep in German, so I used the "neue" here to capture this. @felixcremer what do you think? Remove "neue" insert the subclause or keep as is?

File: 2022/cfp_de.md [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+Wir erwarten von unseren Vortragenden, dass sie sich für die Konferenz registrieren. Da
+die JuliaCon 2022 eine reine Online-Konferenz ist, wird die Anmeldung kostenlos sein.
+
+### Kontakt
+Sie können uns bei allen Fragen und Anliegen unter juliacon@julialang.org erreichen.
+
+# Verhaltenskodex der Konferenz
+JuliaCon ist bestrebt, allen Teilnehmern ein positives Konferenzerlebnis zu
+bieten, unabhängig von Geschlecht, Geschlechtsidentität,
+sexueller Orientierung, Behinderung, körperlichem Aussehen, Körpergröße, Rasse,
```
Comment (Keno) [inline 2022/cfp_de.md:167]: I realize this is a direct translation from the English, but for obvious reasons, the word race has very strong connotations in German not present in English and is not commonly used in discourse. I would perhaps suggest a less literal translation such as "Hautfarbe" (skin color) which is more or less what Americans mean when they use the word race.

File: 2022/cfp_de.md [hunk 3]
```diff
@@ -0,0 +1,330 @@
+# JuliaCon 2022 CFP
+*Dieser Call for Proposals wurde am 2021-04-01 11:59 (UTC) beendet.*
```
Comment (Keno) [inline 2022/cfp_de.md:2]: Call for Proposals -> `Aufruf zur Einreichung von Vorschlägen`?

File: 2022/cfp_de.md [hunk 4]
```diff
@@ -0,0 +1,330 @@
+# JuliaCon 2022 CFP
+*Dieser Call for Proposals wurde am 2021-04-01 11:59 (UTC) beendet.*  
+
+Wir laden Sie dazu ein Vortragsvorschläge für die JuliaCon 2022 einzureichen.
```
Comment (Keno) [inline 2022/cfp_de.md:4]:
```suggestion
Wir laden Sie herzlich dazu ein Vortragsvorschläge für die JuliaCon 2022 einzureichen.
```
still reads a little abrupt though to my taste, maybe some like
```suggestion
Wie jedes Jahr, laden wir Sie auch in diesem Jahr herzlich dazu ein Vortragsvorschläge für die JuliaCon 2022 einzureichen.
```

File: 2022/cfp_de.md [hunk 5]
```diff
@@ -0,0 +1,330 @@
...
+
+Die JuliaCon bietet traditionell eine Mischung von Vortragsthemen für Einsteiger,
```
Comment (Keno) [inline 2022/cfp_de.md:6]:
```suggestion
Die JuliaCon bietet traditionell eine Mischung an Vorträgen sowohl für Einsteiger, als auch
```

File: 2022/cfp_de.md [hunk 6]
```diff
@@ -0,0 +1,330 @@
...
+für Fortgeschrittene, Themen aus verschiedenen Fachrichtungen, von Entwicklern,
```
Comment (Keno) [inline 2022/cfp_de.md:7]:
```suggestion
für Fortgeschrittene, mit Themen aus verschiedenen Fachrichtungen, von Entwicklern,
```

File: 2022/cfp_de.md [hunk 7]
```diff
@@ -0,0 +1,330 @@
...
+Forschern aus der Industrie oder dem akademischen Bereich. Wenn Sie in der
+Vergangenheit mit Julia gearbeitet haben, ist die JuliaCon der beste Ort, um
+Ihre Arbeit mit anderen aus der Juliagemeinschaft zu teilen.
+
+Um einen genaueren Einblick in die üblichen Vortragsthemen zu bekommen,
```
Comment (Keno) [inline 2022/cfp_de.md:12]:
```suggestion
Für einen genaueren Einblick in die üblichen Vortragsthemen,
```

File: 2022/cfp_de.md (hunk 1 above)
Comment (SaschaMann) [reply 2022/cfp_de.md:86]: Perhaps "die [gerade] beigetretenen Teilnehmer"?

**Review by SaschaMann — COMMENTED** [review]
Left some minor suggestions.

---

The heading levels might be one off, a markdown document is usually supposed to have only one top level heading (`#`) though I don't know what the standards are in this repo.

The Siezen feels out of place for JuliaCon and the Julia community, where usually people use first names and less formal language. E.g. I probably wouldn't start using "Sie" in a German channel on the JuliaCon Discord and #deutsch on Zulip uses "du" as well. But I'm not sure what's best for the CfP.

File: 2022/cfp_de.md [hunk 8]
```diff
@@ -0,0 +1,330 @@
...
+stellen. Einem Minisymposium wird ein Zeitfenster von zwei bis drei Stunden zugewiesen.
+*Hinweis:* Im Gegensatz zur JuliaCon 2021 werden die Minisymposia vor dem offiziellen
+Beginn der Konferenz, also zusammen mit den Workshops stattfinden.
+
+### Workshops
+Bei einem Workshop handelt es sich in der Regel um eine dreistündige
+interaktive praktische Veranstaltung, bei welcher der Vortragende live vor dem Publikum
+programmiert und den Code mit den Teilnehmern teilt, sodass diese leicht
+folgen können.  Die Workshops finden vor dem offiziellen Beginn der Konferenz
+statt (wie auch die Minisymposia). Zu den Themen dieser Schiene gehören beispielsweise
```
[the remark this answers is not in the document]
Comment (mfherbst) [reply 2022/cfp_de.md:75]: I'll use "Kategorie" I used that else where as well.

File: 2022/cfp_de.md [hunk 9]
```diff
@@ -0,0 +1,330 @@
...
+- In der zweiten Auswahlphase entscheidet der Programmausschuss anhand der
+  Bewertungen der Gutachter, der Kommentare und der Neueinstufungen über die
+  endgültige Annahme der Vorschläge. Abgelehnte Vorschläge werden im Hinblick
+  auf eine Neueinstufung diskutiert, bevor eine vollständige Ablehnung erfolgt.
+
+- Nachdem alle Vorschläge klassifiziert worden sind, werden Annahmen und
+  Ablehnungen veröffentlicht.
+
+- Ein spezielles Symposium ist reserviert für Blitzvorträge von
+  Google-Summer-of-Code-Studenten. Da die Genehmigung/Ablehnung von
```
[the remark this answers is not in the document]
Comment (mfherbst) [reply 2022/cfp_de.md:313]: :+1:

File: 2022/cfp_de.md (hunk 1 above)
Comment (mfherbst) [reply 2022/cfp_de.md:86]: I like that. I used something along this line now.

File: 2022/cfp_de.md (hunk 3 above)
Comment (mfherbst) [reply 2022/cfp_de.md:2]: why not. Avoids the Anglicism ;).

File: 2022/cfp_de.md (hunk 7 above)
Comment (mfherbst) [reply 2022/cfp_de.md:12]: I restructured the text a bit here to make it overall hopefully clearer.

**Comment by mfherbst** [comment]
> The Siezen feels out of place for JuliaCon and the Julia community, where usually people use first names and less formal language. E.g. I probably wouldn't start using "Sie" in a German channel on the JuliaCon Discord and #deutsch on Zulip uses "du" as well. But I'm not sure what's best for the CfP.

I agree with this concern. In fact along similar arguments I would have gone for "du" as well. We discussed this a bit also wrt. other non-English languages (e.g. Russian, French also have two forms). For now we decided to go uniformly towards the formal, just because too formal seems more appropriate for a call, which is potentially also read by people from outside the community.

**Comment by matbesancon** [comment]
FYI I've also used Siezen in the French CFP

File: 2022/cfp_de.md (hunk 2 above)
Comment (Keno) [reply 2022/cfp_de.md]: Removal sounds good to me.

**Comment by Keno** [comment]
Formal address seems appropriate for a professional cfp.

File: 2022/cfp_de.md [hunk 10]
```diff
@@ -102,25 +103,25 @@ Universitätskurs, oder ganz allgemein wie Julia sich gut in den persönlichen
...
 keine Zeit für Fragen vorgesehen ist, ermutigen wir Sie im Besonderen
 Kontaktinformationen wie Ihren Twitter- oder Github-Nutzernamen
-oder Ihre E-Mail-Adresse auf Ihren Folien anzugeben, damit Zuseher Ihnen
+oder Ihre E-Mail-Adresse auf Ihren Folien anzugeben, damit Zuseher:innen Ihnen
 Fragen zukommen lassen können. Wir freuen uns besonders, wenn Julianeulinge
 mittels dieser Beitragskategorie ihre Eindrücke mit der Juliagemeinschaft teilen.
-Falls Sie als Lehrer oder Professor Julia in Ihren Kursen verwenden, möchten
-wir Sie bitten, Ihre Studenten einzuladen, bei dieser Kategorie Ihre Erfahrungen
+Falls Sie als Lehrer:in oder Professor:in Julia in Ihren Kursen verwenden, möchten
+wir Sie bitten, Ihre Student:innen einzuladen, bei dieser Kategorie Ihre Erfahrungen
```
Comment (SaschaMann) [inline 2022/cfp_de.md:110]: Studierende is well-established, even if Student:innen is more consistent with the rest.

File: 2022/cfp_de.md [hunk 11]
```diff
@@ -102,25 +103,25 @@ Universitätskurs, oder ganz allgemein wie Julia sich gut in den persönlichen
...
 mit der Juliagemeinschaft zu teilen.
 
 # Details zur Einreichung und Begutachtung
 Wir verwenden ein anonymisiertes Begutachtungsverfahren um Auswahlverzerrungen
-in Bezug auf den Sprecher zu vermeiden. Obwohl ein Doppelblindverfahren nicht
-vollständig gewährleistet werden kann, da die meisten Einreichungen auf öffentlichen
+in Bezug auf den Sprecher / die Sprecherin zu vermeiden. Obwohl ein Doppelblindverfahren
```
Comment (SaschaMann) [inline 2022/cfp_de.md:115]: This is a bit awkward in singular with a specific article, perhaps "die sprechende Person" or "die vortragende Person" works better?

File: 2022/cfp_de.md [hunk 12]
```diff
@@ -162,47 +163,47 @@ die JuliaCon 2022 eine reine Online-Konferenz ist, wird die Anmeldung kostenlos
...
-Nachfolgend finden Sie die Richtlinien und Verfahren, welche die Gutachter
+Nachfolgend finden Sie die Richtlinien und Verfahren, welche die Gutachter:innen
 bei der Beurteilung Ihrer Einreichung anwenden werden.
-Die Hauptaufgabe der Gutachter besteht darin, die Qualität der Inhalte
+Die Hauptaufgabe der Gutachter:innen besteht darin, die Qualität der Inhalte
 auf der JuliaCon zu gewährleisten.
 
 ## Interessenkonflikte
-Im Falle eines Interessenkonflikts verpflichtet sich der Gutachter, sich von
+Im Falle eines Interessenkonflikts verpflichtet sich der Begutachtende, sich von
```
Comment (SaschaMann) [inline 2022/cfp_de.md:189]: Same as above: "begutachtende Person" avoids the "der:die"/"der/die" for singular

File: 2022/cfp_de.md [hunk 13]
```diff
@@ -162,47 +163,47 @@ die JuliaCon 2022 eine reine Online-Konferenz ist, wird die Anmeldung kostenlos
...
 der Begutachtung zurückzuziehen und den Organisatoren mitzuteilen,
-dass ein Ersatz gefunden werden muss. Kein Gutachter sollte einen Vortrag
-begutachten, an dem er als Autor beteiligt ist oder in einem anderen
+dass ein Ersatz gefunden werden muss. Kein:e Gutachter:in sollte einen Vortrag
+begutachten, an dem er/sie als Autor:in beteiligt ist oder in einem anderen
 Interessenkonflikt steht. In den [PNAS-Richtlinien](https://www.pnas.org/page/authors/conflict-of-interest)
 (auf Englisch) finden Sie eine Definition und Beispiele. Ein solcher Konflikt
 ist insbesondere dann geboten, wenn die begutachtete Arbeit oder die beteiligten
-Autoren in einer Weise mit dem Gutachter verbunden sind,
+Autoren in einer Weise mit dem/der Gutachter:in verbunden sind,
```
Comment (SaschaMann) [inline 2022/cfp_de.md:196]: See above. Or "dem:der" to stick to the ":" form

File: 2022/cfp_de.md [hunk 14]
```diff
@@ -162,47 +163,47 @@ die JuliaCon 2022 eine reine Online-Konferenz ist, wird die Anmeldung kostenlos
...
 "die einen finanziellen oder intellektuellen Interessenkonflikt
 darstellt oder als solchen wahrgenommen werden könnte" (PNAS-Richtlinien oben).
 
 ## Verhaltensregeln
-Der/die Gutachter/in verpflichtet sich, den [Verhaltenskodex](https://juliacon.org/2022/coc)
+Der/die Gutachter:in verpflichtet sich, den [Verhaltenskodex](https://juliacon.org/2022/coc)
```
Comment (SaschaMann) [inline 2022/cfp_de.md:201]: See above.

File: 2022/cfp_de.md [hunk 15]
```diff
@@ -162,47 +163,47 @@ die JuliaCon 2022 eine reine Online-Konferenz ist, wird die Anmeldung kostenlos
...
 (auf Englisch) der Konferenz bei der Bewertung und bei allen Mitteilungen
 während des Begutachtungsprozesses zu lesen und einzuhalten.
 
 Wenn eine eingereichte Zusammenfassung nicht mit dem Verhaltenskodex
-übereinstimmt, sollte der Gutachter dies dem Organisationskomitee anzeigen.
+übereinstimmt, sollte der/die Gutachter:in dies dem Organisationskomitee anzeigen.
```
Comment (SaschaMann) [inline 2022/cfp_de.md:206]: See above.

File: 2022/cfp_de.md [hunk 16]
```diff
@@ -224,16 +225,16 @@ Die Nichteinhaltung dieser Kriterien führt zu einer niedrigeren Bewertung des B
...
      - Wissenschaftliches Rechnen
-     - Bewährte Praktiken beim Software-Engineering
+     - Bewährte Praktiken in der Softwareentwicklung
      - Statistik, maschinelles Lernen und Künstliche Intelligenz
      - Computergestützte Geistes- und Sozialwissenschaften
 
 4. Falls das vom Autor vorgeschlagene Beitragsformat (Experience,
    Kurzvortrag, Vortrag, Workshop, Poster, Minisymposium, Birds of a Feather)
-   unpassend erscheint, kann der Gutachter dies anzeigen und ein anderes Format
+   unpassend erscheint, kann der/die Gutachter:in dies anzeigen und ein anderes Format
```
Comment (SaschaMann) [inline 2022/cfp_de.md:237]: See above.

File: 2022/cfp_de.md (hunk 10 above)
Comment (mfherbst) [reply 2022/cfp_de.md:110]: Classic copy n' paste. I caught it somewhere though ...

File: 2022/cfp_de.md (hunk 11 above)
Comment (mfherbst) [reply 2022/cfp_de.md:115]: That sounds very strange. I'll go for "den Vortragenden".

File: 2022/cfp_de.md (hunk 12 above)
Comment (mfherbst) [reply 2022/cfp_de.md:189]: Same thing, I think introducing "Person" is a bit strange. What we have now sounds more natural, no?

**Comment by mfherbst** [comment]
@SaschaMann Thanks for your comments. I changed the formulation at the places you flagged. Feel free to suggest differently if you have other ideas for improvement :smile:.

File: 2022/cfp_de.md (hunk 12 above)
Comment (SaschaMann) [reply 2022/cfp_de.md:189]: Rewriting it with Person is quite common ime. "der Begutachtende" turns it male again.

Comment (SaschaMann) [reply 2022/cfp_de.md:189]: That works best, I think. Maybe that works for Vortragende too in the places where it's used in singular?

Comment (SaschaMann) [reply 2022/cfp_de.md:189]: That's the last nitpick I have, otherwise it looks good :)

Comment (mfherbst) [reply 2022/cfp_de.md]: Seems to work rather well. Thanks for your help @SaschaMann.

**Pull request merged**